fastapi-google-stt/app/services/log_script_service.py

```
@@ -45,17 +45,58 @@ async def get_scripts_by_user_id(db: AsyncSession, user_id: int) -> list | None:
+        raise ValueError(f"Script with ID {script_id} not found")
+
+    # 파일 경로 가져오기
+    file_path = script.file_path
+
+    # 파일 내용 읽기
+    if os.path.exists(file_path):
+        try:
+            with open(file_path, "r", encoding="utf-8") as file:
+                return file.read()  # 파일 내용 반환
+        except Exception as e:
+            raise IOError(f"Failed to read file {file_path}: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)